package-lock.json

```diff
@@ -3938,9 +3938,9 @@
       }
     },
     "@tacc/core-styles": {
-      "version": "git+https://git@github.com/TACC/Core-Styles.git#16c8873586785344ebd84b8360584c5a390bc499",
+      "version": "git+https://git@github.com/TACC/Core-Styles.git#115bcdc07bfbf8b97ad804f8e36ffc672234d330",
       "dev": true,
-      "from": "@tacc/core-styles@git+https://git@github.com/TACC/Core-Styles.git",
+      "from": "@tacc/core-styles@git+https://git@github.com/TACC/Core-Styles.git#bugfix/fp-1542-figure-image-margin",
```

Comment:
**Warning.** Do not commit the branch URL to this file. Re install/lock after merge of https://github.com/TACC/Core-Styles/pull/25.